frontend/src/components/SamplesList.tsx

```diff
@@ -72,9 +72,6 @@ export default function SamplesList({
     useSamplesListQuery({
       variables: {
         where: parentWhereVariables || {},
-        options: {
-          limit: MAX_ROWS,
-        },
```

Comment:
Remove this limit (500 rows) as we need to fetch all samples and then sort them in the GraphQL server. Note that ultimately we'll still return 500 samples to the user.